backend/dropbox/dropbox.go

```diff
@@ -247,7 +247,13 @@ folders.`,
 			Help:     "Specify a different Dropbox namespace ID to use as the root for all paths.",
 			Default:  "",
 			Advanced: true,
-		}}...), defaultBatcherOptions.FsOptions("For full info see [the main docs](https://rclone.org/dropbox/#batch-mode)\n\n")...),
+		}, {
+			Name:     "export_formats",
+			Help:     "Comma separated list of preferred formats for exporting files",
+			Default:  "",
```

Comment:
Define the Default as `fs.CommaSepList{}` and the config parser will parse it for you into a `[]string`